### PR TITLE
Relative dates not appearing in moreons

### PR DESCRIPTION
**Pull request by matthew-andrews**
@ironsidevsquincy

### CODE REVIEW

File: client/_mixins.scss [hunk 1]
```diff
@@ -1,12 +1,6 @@
 @mixin filter($funcs...) {
-	-webkit-filter: $funcs;
-	-moz-filter: $funcs;
-	-o-filter: $funcs;
-	-ms-filter: $funcs;
```
Comment (ironsidevsquincy) [inline client/_mixins.scss]: definitely not needed? e.g latest chrome only works with `webkit` prefix

Comment (matthew-andrews) [inline client/_mixins.scss]: origami-build-tools ships with autoprefixer

Comment (matthew-andrews) [inline client/_mixins.scss]: > autoprefixerBrowsers: Array An array of strings of browser names for autoprefixer to check what prefixes it needs. (Default: ["> 1%", "last 2 versions", "ie > 6", "ff ESR"])

Comment (ironsidevsquincy) [inline client/_mixins.scss]: a-mazing

**Pull request merged**